challenges/test/lib/src/test/java/code/treetest/TreeTest.java

```diff
@@ -169,4 +170,41 @@ public void binarySearchTreeTest(){
 
     assertEquals(1234,newTree.findMaximunValue());
   }
+
+  @Test
+  public void treeFizzBuzz(){
+    Tree newTree = new Tree();
+    Node a = new Node(2);
+    Node b = new Node(7);
+    Node c = new Node(5);
+    Node d = new Node(15);
+    Node e = new Node(6);
+    Node f = new Node(5);
+    Node g = new Node(11);
+    Node h = new Node(9);
+    Node i = new Node(4);
+//    Node j = new Node(2);
+    Node k = new Node(5);
+
+    newTree.root=a;
+
+    a.left=b;
+    b.left=d;
+    b.right=e;
+    e.left=f;
+    e.right=g;
+
+    a.right=k;
+    k.right=h;
+    h.left=i;
+    i.right=new Node(144);
+
+    Tree fizzTree =FizzBuzzTree.fizzBuzzTree(newTree);
+
+    System.out.println(fizzTree.root.value);
+    System.out.println(fizzTree.root.left.value);
+    System.out.println(fizzTree.root.right.value);
+
+    assertEquals("FizzBuzz",fizzTree.root.left.left.value);
```

Comment:
I would have liked to see you testing full functionality of the method you wrote, there is no verification that Fizz or Buzz work in this test.